Allow to enable/disable colors when colors.js is imported as an ES6 module

I am using typescript and this code:
```typescript
import * as colors from "colors"; 

colors.enabled = true;
```
give me this error:

    error TS2540: Cannot assign to 'enabled' because it is a constant or a read-only property    

As explained in https://github.com/Microsoft/TypeScript/issues/6751, there are workarounds but the right way could be to expose methods to enable/disable colors.

If it sounds relevant to you, I would be glad to help.

--- a/index.d.ts
+++ b/index.d.ts
@@ -46,6 +46,8 @@ export interface Color {
     zalgo: Color;
 }
 
+export function enable(): void;
+export function disable(): void;
 export function setTheme(theme: any): void;
 
 export let enabled: boolean;

--- a/lib/colors.js
+++ b/lib/colors.js
@@ -2,7 +2,7 @@
 
 The MIT License (MIT)
 
-Original Library 
+Original Library
   - Copyright (c) Marak Squires
 
 Additional functionality
@@ -41,6 +41,14 @@ colors.supportsColor = require('./system/supports-colors').supportsColor;
 if (typeof colors.enabled === "undefined") {
   colors.enabled = colors.supportsColor() !== false;
 }
+
+colors.enable = function () {
+  colors.enabled = true;
+};
+
+colors.disable = function () {
+  colors.enabled = false;
+};
 
 colors.stripColors = colors.strip = function(str){
   return ("" + str).replace(/\x1B\[\d+m/g, '');

--- a/safe.d.ts
+++ b/safe.d.ts
@@ -4,6 +4,8 @@
 // Definitions: https://github.com/Marak/colors.js
 
 export const enabled: boolean;
+export function enable(): void;
+export function disable(): void;
 
 export function strip(str: string): string;
 export function stripColors(str: string): string;

--- a/safe.d.ts
+++ b/safe.d.ts
@@ -6,6 +6,7 @@
 export const enabled: boolean;
 export function enable(): void;
 export function disable(): void;
+export function setTheme(theme: any): void;
 
 export function strip(str: string): string;
 export function stripColors(str: string): string;

